### ISSUE TITLE
ZooKeeper 3.9.2 bin file does not exist any more

### ISSUE THREAD

**Issue by hongbo-miao** [sleighzy/ansible-zookeeper#54]
Currently

```yaml
- hosts: zookeeper-nodes
  roles:
    - role: sleighzy.zookeeper
```

gives error

```
TASK [sleighzy.zookeeper : Download Apache ZooKeeper] **************************
fatal: [integrated-test-group-server]: FAILED! => {"changed": false, "dest": "/tmp", "elapsed": 0, "gid": 0, "group": "root", "mode": "01777", "msg": "Request failed", "owner": "root", "response": "HTTP Error 404: Not Found", "size": 32768, "state": "directory", "status_code": 404, "uid": 0, "url": "https://dlcdn.apache.org/zookeeper/zookeeper-3.9.2/apache-zookeeper-3.9.2-bin.tar.gz"}
```

Seems 

https://dlcdn.apache.org/zookeeper/

3.9.2 is gone, but 3.9.3 is there.

It is worth using 3.9.3 by default. ☺️

**Comment by sleighzy** [comment]
Fixed in version 1.3.3 by #55

**Comment by sleighzy** [comment]
Thanks @hongbo-miao , this has been updated and released now.
